mne_bids/mne_bids.py

```diff
@@ -29,7 +29,7 @@
                     make_dataset_description, _write_json,
                     _read_events, _mkdir_p, age_on_date,
                     copyfile_brainvision, copyfile_eeglab,
```

Comment:
@monkeyman192 these functions are not affected by `write_mode` ?

Reply:
oh yeah, good catch. All these lines here will need to be updated

https://github.com/mne-tools/mne-bids/blob/11465ad16a234f528cc9137e6373824a5008bcb2/mne_bids/mne_bids.py#L752-L761

Reply:
actually, I just realised that we don't actually have to modify these functions. Their destinations are `raw_file_bids` which we already check for existence further up

https://github.com/mne-tools/mne-bids/blob/11465ad16a234f528cc9137e6373824a5008bcb2/mne_bids/mne_bids.py#L736

(This line is going to change anyway as I am going to remove the `_check_file_exists` function and put back what we had before...